app/models.py

```diff
@@ -786,17 +817,17 @@ class ServiceSmsSender(db.Model):
     def get_reply_to_text(self):
         return try_parse_and_format_phone_number(self.sms_sender)
 
-    def serialize(self):
-        return {
-            "id": str(self.id),
-            "sms_sender": self.sms_sender,
-            "service_id": str(self.service_id),
-            "is_default": self.is_default,
-            "archived": self.archived,
-            "inbound_number_id": str(self.inbound_number_id) if self.inbound_number_id else None,
-            "created_at": self.created_at.strftime(DATETIME_FORMAT),
-            "updated_at": get_dt_string_or_none(self.updated_at),
-        }
+    def serialize(self) -> SerializedServiceSmsSender:
+        return SerializedServiceSmsSender(
+            id=str(self.id),
+            sms_sender=self.sms_sender,
+            service_id=str(self.service_id),
+            is_default=self.is_default,
+            archived=self.archived,
+            inbound_number_id=str(self.inbound_number_id) if self.inbound_number_id else None,
+            created_at=self.created_at.strftime(DATETIME_FORMAT) if self.created_at else None,
```

Comment:
This logic can be kept as `self.created_at.strftime(DATETIME_FORMAT)` since `created_at` is not nullable.